### ISSUE TITLE
Fix wipeout behaviour when deleting Postgres

### ISSUE THREAD

**Issue by soosap** [kubedb/project#255]
**Replicate**

**Step 1**: I deploy the following `Postgres` workload

```yaml
apiVersion: kubedb.com/v1alpha1
kind: Postgres
metadata:
  name: registry-postgres
  namespace: default
  labels:
    app: postgres
    chart: "postgres-0.1.0"
    release: registry
    heritage: Tiller
spec:
  version: "10.2"
  replicas: 1
  doNotPause: True
  databaseSecret:
    secretName: registry-postgres
  storage:
    storageClassName: aws-ebs-gp2
    accessModes:
    - ReadWriteOnce
    resources:
      requests:
        storage: 1Gi
  archiver:
    storage:
      storageSecretName: s3-secret
      s3:
        endpoint: s3.amazonaws.com
        bucket: my-backup-bucket
  init:
    postgresWAL:
      storageSecretName: s3-secret
      s3:
        endpoint: s3.amazonaws.com
        bucket: my-backup-bucket
        prefix: "kubedb/default/registry-postgres/archive"
      pitr: latest # <==============================
serviceTemplate:
  metadata:
    name: registry-postgres
    namespace: default
    labels:
      app: postgres
      chart: "postgres-0.1.0"
      release: registry
      heritage: Tiller
  spec:
    type: ClusterIP
```

**Step 2:** I decide to start from scratch and avoid the previously taken backups
```sh
$ kubectl patch -n default pg/registry-postgres -p '{"spec":{"doNotPause":false}}' --type="merge"

$ kubectl delete -n default pg/registry-postgres

$ kubectl patch -n default drmn/registry-postgres -p '{"spec":{"wipeOut":true}}' --type="merge"

$ kubectl delete -n default drmn/registry-postgres
```

**Step 3:** I realise that the basebackup and WAL files still exist in the S3 bucket. So I manually delete all files under that particular prefix, i.e. `my-backup-bucket/kubedb/default/registry-postgres`

**Step 4:** I re-deploy the same workload as specified in Step 1.

**Expected**
I expect an empty state database as no more basebackup and WAL files exist in the bucket from which a recovery could be initiated.

**Actual**
It recovers to the latest point in time as if the WAL and basebackup files would still be present in the S3 bucket. It means those files must still be available from a cache or something like that.

**Comment by the-redback** [comment]
Fixed in https://github.com/kubedb/postgres/pull/195. Now walG data also wipedout when `wipeOut` is peformed.

**Comment by tamalsaha** [comment]
Fixed in 0.9.0-rc.0
